docs/integrations/sources/persistiq.md

```diff
@@ -0,0 +1,38 @@
+# Chartmogul
```

Comment:
```suggestion
# PersistIq
```
🤔

Reply:
Busted, I took the one from the Chartmogul source but didn't to a good job at copying 😅